features/location.yml

```diff
@@ -0,0 +1,22 @@
+name: Location
+description: "The `Location` global object represents the current address of the web page loaded in the browser. It makes all component parts of the address accessible and can be manipulated to initiate navigations."
+spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#the-location-interface
+# `ancestorOrigins` makes this baseline false but is not required for usage of the feature.
+status:
+  compute_from: api.Location
+compat_features:
```

Comment:
What would this look like with `api.Window.location` and `api.Document.location`? I _think_ those are going to be the most interesting uses of the `Location` API and the impetus for this API to even exist.

Reply:
Argh, I forgot the actual implementation. Again. Adding a full search on feature strings and substrings to my pre-commit checklist.

Reply:
Ok baseline remains same